frontend/src/components/DropboxChooser/components/Content.tsx

```diff
@@ -13,7 +13,7 @@ const Chooser: FC = () => {
     dropins();
   }, []);
 
-  return <div id={ID} data-app-key={APP_KEY}></div>;
+  return <div id={ID} data-app-key={APP_KEY} style={{ display: "none" }}></div>;
```

Comment:
Oh yes! https://github.com/chanzuckerberg/corpora-data-portal/pull/1029/files#r605167167 

```
export const Wrapper = styled.div`
  ...
  & > :not(:last-child) { // <---------- this is targeting every element, which included the empty div, so there was 32px margin instead of the expected 16px because of it!
    ...
  }
`
```